specutils/io/default_loaders/chandra_hetg.py

```diff
@@ -0,0 +1,54 @@
+from astropy.io import fits
+from astropy.units import Unit
+
+from specutils.io.registers import data_loader
+from specutils import XraySpectrum1D
+
+__all__ = ['chandra_hetg_identify', 'hetg_spectrum_loader']
+
+def chandra_hetg_identify(origin, *args, **kwargs):
+    """Check whether given file contains Chandra HETG spectral data."""
+    with fits.open(args[0]) as hdu:
```

Comment:
This will fail catastrophically if the file it's called on it not a fits file, but e.g. a txt file. Yes, Chandra files will always be fits file, but the "identify" function should work on any file to *identify* if this is a Chandra file. See discussion in #404

Reply:
I mean, what if I have a txt file that for some weird reason ends on "fit", let's say "best_model.fit"?

Reply:
Can you write me a solution for that?

Reply:
Sure. I think https://github.com/astropy/specutils/blob/master/specutils/io/default_loaders/hst_cos.py#L13 would work. Essentially, that just wraps your check on HDU keywords in the try-except block, which will take care of non-fits files.

However, I now wonder, why you you want to identify HETG spectra? Since ARF, RMF and PHA (TYPE I or II) are all OGIP standardized, would it not make more sense to have an "identify OGIP complient spectrum" identifier and then get all X-ray spectra (XMM, Chandra, ACIS, HETG, SUSAku,...) for the price of one?